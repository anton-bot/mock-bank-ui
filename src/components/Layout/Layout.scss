.Layout {
  padding: 1em;

  .main-content {
    max-width: 800px;
    margin: 1em auto;
  }
}
