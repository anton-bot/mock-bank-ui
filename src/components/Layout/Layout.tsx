import './Layout.scss';
import { PageHeader } from '../PageHeader/PageHeader';

export const Layout: React.FC = (props) => (
  <div className="Layout">
    <PageHeader />
    <div className="main-content">{props.children}</div>
  </div>
);
