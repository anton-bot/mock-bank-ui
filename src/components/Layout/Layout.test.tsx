import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Layout } from './Layout';

it('matches snapshot', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Layout>Sample text</Layout>
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
