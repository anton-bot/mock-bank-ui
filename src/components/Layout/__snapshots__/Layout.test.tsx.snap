// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Layout"
>
  <div
    className="PageHeader"
  >
    <div>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt=""
          className="Logo"
          src="bankone.png"
        />
      </a>
    </div>
    <div>
      <div
        className="SiteMenu collapsed"
      >
        <div
          className="hamburger"
          onClick={[Function]}
        >
          â˜°
        </div>
        <div
          className="menu-items"
        >
          <a
            href="/app/accounts"
            onClick={[Function]}
          >
            Accounts
          </a>
          <a
            href="/app/transfer"
            onClick={[Function]}
          >
            Transfer Money
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="main-content"
  >
    Sample text
  </div>
</div>
`;
